doc/classes/EditorFileSystem.xml

```diff
@@ -84,6 +84,12 @@
 				Emitted if a resource is reimported.
 			</description>
 		</signal>
+		<signal name="resources_reimporting">
```

Comment:
Why is the signal called `resources_reimporting`, but description mentions one resource?